tests/test_diaForcedSource.py

```diff
@@ -0,0 +1,207 @@
+# This file is part of ap_association.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import unittest
+
+from lsst.afw.cameraGeom.testUtils import DetectorWrapper
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.image.utils as afwImageUtils
+import lsst.afw.table as afwTable
+import lsst.daf.base as dafBase
+import lsst.meas.algorithms as measAlg
+from lsst.ap.association import \
+    DiaForcedSourceTask, \
+    make_dia_object_schema
+import lsst.utils.tests
+
+
+def create_test_dia_objects(n_points, wcs, startPos=100):
+    """Create dummy DIASources or DIAObjects for use in our tests.
+    Parameters
+    ----------
+    n_points : `int`
+        Number of DiaObject test points to create.
+    wcs : `lsst.afw.geom.SkyWcs`
+        Wcs to convert RA/Dec to x/y if provided.
```

Comment:
What do you mean by "if provided" here? It doesn't look optional.

Reply:
Removed. I re-purposed a previous method.